packages/core/src/document.ts

```diff
@@ -418,6 +418,13 @@ class Document extends EventEmitter {
       // If they have the same block number fall through to fallback mechanism
     }
 
+    // The anchor states are the same for both logs. Compare log lengths and choose the one with longer length.
+    if (state1.log.length > state2.log.length) {
```

Comment:
I think this case is backwards. We're supposed to return true when selecting state2's log, and false when selecting state1's log

Reply:
👍